tasks/taglib.rb

```diff
@@ -0,0 +1,37 @@
+# frozen-string-literal: true
+
+class Taglib
+  class << self
+    def plat
+      ENV['PLATFORM'] || 'i386-mingw32'
+    end
+
+    def version
+      ENV['TAGLIB_VERSION'] || '1.9.1'
```

Comment:
@robinst so it turns out that Swig fails to run against TagLib 1.9.1 because https://github.com/robinst/taglib-ruby/pull/103 added a dependency on TagLib 1.10 header files.

If I try to regenerate it creates a big mess so I held off on that for now.

Reply:
Hmm I see CI uses 1.11.1 so maybe I should just bump to that? I still don't like regenerating all the wrappers as a side effect of a tooling change.

Reply:
OK never mind the current wrap files are 1.11.1-clean.